README.md

```diff
@@ -45,6 +50,18 @@ It is conceivable that some extensions may not be compatible with other extensio
 
 Because OCFL clients are not required to implement any or all extensions, it is also possible that a client may encounter an extension that it does not implement. In these cases, it is up to the client to decide how to proceed. A client may fail on unsupported extensions, or it may choose to ignore the extensions and carry on.
 
+### Optional Extension Initializer
+
+In order to help address some of the [undefined behaviors](#undefined-behavior) as well as to provide a specified approach for allowing an OCFL client to answer questions such as:
+  - Is an extension deactivated, only applying to earlier versions of the object?
+  - Should extensions be loaded in a specific order?
```

Comment:
Agreed, a lot of assumptions about implementation and execution here. I would prefer something like "config" rather than "init" which is more neutral in its interpretation.

Reply:
Resolved but also second instance -> https://github.com/OCFL/extensions/pull/36#pullrequestreview-528349733

Reply:
But also `init` vs `config` question

Reply:
I agree with @neilsjefferies that `config` would be preferable (quite separate from jokes about saying `init` and sounding like a shorthand for "isn't it")

Reply:
I am fine with changing away from `init`. I am a little concerned about overloading "config" since we already have [Configuration Files](https://github.com/OCFL/extensions/blob/master/README.md#configuration-files).

This pull-request introduces a new section named "[Optional Extension Initializer](https://github.com/OCFL/extensions/blob/555e8c683d947d0542a2fb6c3f97d6cdb600f694/README.md#optional-extension-initializer)". I think having "Optional Extension Configuration" would be confusing.

Instead of either `init` or `config`, how does `entry` sound, i.e "Optional Extension Entrypoint"? Although I am certainly open to other suggestions.

Reply:
I agree with @awoods concerns about `config`. I personally prefer `init`, hearkens back to good old `init.d`, but don't feel strongly about it, or have any better suggestions currently.

Reply:
Hmmm... I think the change of section title just brings the problem to a higher level. How about calling the file `initial_extension` and the section **Optional Initial Extension**

Reply:
Thanks, @zimeon .
Are you suggesting this extension's subdirectory be named `initial_extension` vs either `init` or `config`? ..such that, the end result would be an extension directory like:
```
[storage-root]/extensions/initial_extension/config.json
```

Reply:
Yes. However, now I look at it I think just `initial` would be clear an avoid the possible execution implications:
```
.../extensions/initial/config.json
```
still with the section name **Optional Initial Extension**

Reply:
This should now be resolved.

I will note, given the shift in phrasing, there may be interest in changing [this line](https://github.com/OCFL/extensions/pull/36/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R58) (or not).